climpred/bootstrap.py

```diff
@@ -173,81 +212,133 @@ def bootstrap_perfect_model(ds,
         init.append(
             compute_perfect_model(
                 smp_ds,
-                control,
                 metric=metric,
-                comparison=comparison,
-                running=running,
-                reference_period=reference_period))
-        if compute_uninitialized_skill:
```

Comment:
no userflow bools anymore, just calculate all

Reply:
Great, we might want to profile this at some point since there's a bunch of steps. Maybe dask can be brought in or something. But that's out of the scope of this PR.